### DISCUSSION TITLE
cpu use

### DISCUSSION THREAD

**Discussion by AdamM2024** [lllyasviel/Fooocus#2420] [Q&A]
were do i put always cpu in fooocus? to us

**Comment by poor7** [comment]
You will need to have about 40 GB of RAM and the generation will be very slow.

**Reply by AdamM2024** [reply]
i have that

**Reply by poor7** [reply]
Use `--always-cpu [CPU_NUM_THREADS]`

**Reply by mashb1t** [reply]
@AdamM2024 depending on your system you either have to edit the run.bat and add `--always-cpu` after entrypoint_with_update.py or use it as additional argument when calling the entrypoint directly.
If you want to optionally specify the amount of CPU cores to allocate for Fooocus then you can also do so with e.g. `--always-cpu 4` for 4 cores, but it's not mandatory.

**Reply by AdamM2024** [reply]
were do i find it edit run file thats with anime?
